### PR TITLE
Update docker instructions

### PR DESCRIPTION
**Pull request by spham-amzn**
Update docker readme instructions to specify instructions to use `LaunchNavStack.bash` for both `o3de_robot_vacuum_navstack` as well as `o3de_robot_vacuum_simulation`.

Fixes https://github.com/o3de/RobotVacuumSample/issues/13 

Signed-off-by: Steve Pham <82231385+spham-amzn@users.noreply.github.com>

### CODE REVIEW

File: Docker/README.md [hunk 1]
```diff
@@ -61,12 +61,20 @@ Then launch the built simulation docker image with the following command
...
-Once the simulation is up and running, launch the robot application docker image, which will bring up RViz to control the robot.
+Once the simulation is up and running, launch the navigation stack inside the simulation docker image, which will bring up RViz to control the robot.
 
 ```
-docker run --rm --network="bridge" --gpus all -e DISPLAY=:1 -v /tmp/.X11-unix:/tmp/.X11-unix -it o3de_robot_vacuum_navstack:latest /data/workspace/LaunchNavStack.bash
+docker run --rm --network="bridge" --gpus all -e DISPLAY=:1 -v /tmp/.X11-unix:/tmp/.X11-unix -it o3de_robot_vacuum_simulation:latest /data/workspace/LaunchNavStack.bash
+
+```
+
+If you created a separate docker image 'o3de_robot_vacuum_navstack:latest' which only contains the navigation stack and Rviz2, you can launch it using that image, provided that the simulation docker image 'o3de_robot_vacuum_simulation' is running.
```
Comment (hultonha) [inline Docker/README.md:71]: uber nit: It would be good to be consistent on naming of `RViz` (is it `RViz`, `Rviz` or `rviz`?)

**Review by hultonha — APPROVED** [review]
LGTM just one super minor comment

File: Docker/README.md (hunk 1 above)
Comment (spham-amzn) [reply Docker/README.md:71]: I've seen `rviz`, `RViz`, and `RVIZ`. I'll use RViz

**Review by smurly — APPROVED** [review]
great improvement

**Pull request merged**